lib/helper/WebDriverIO.js

```diff
@@ -992,18 +992,23 @@ class WebDriverIO extends Helper {
   /**
    * {{> ../webapi/waitForVisible }}
    */
-  waitForVisible(locator, sec) {
+  waitForVisible(locator, sec, reverse) {
```

Comment:
please don't use boolean inside helper methods. Especially if they transform the behavior in the opposite way. It is unclear that `wait for visible "#user", 10, true` is actually waits for invisible.